frontend/src/text_resources.js

```diff
@@ -786,11 +789,26 @@ let LOCALIZE = new LocalizedStrings({
     emibTest: {
       //Home Page
       homePage: {
+        testTitle: "La test pratique de BRG-e",
         welcomeMsg: "Bienvenue dans le test pratique de BRG-e"
       },
 
       //HowTo Page
       howToPage: {
+        introductionPage: {
+          title: "Aperçu général",
```

Comment:
see previous comment

Reply:
Removed in commit #https://github.com/code-for-canada/project-thundercat/pull/435/commits/1f893d658222afa311f49821494f9f6bc659ffae